components/log-viewer-webui/client/src/pages/SearchPage/SearchControls/search-requests.ts

```diff
@@ -5,7 +5,7 @@ import {
     QueryJobSchema,
     submitQuery,
 } from "../../../api/search";
-import useSearchStore from "../SearchState/index";
+import useSearchStore, { SEARCH_STATE_DEFAULT } from "../SearchState/index";
```

Comment:
```suggestion
import useSearchStore, {SEARCH_STATE_DEFAULT} from "../SearchState";
```
though i didn't test on my end